platform-api/src/routers/platform.py

```
@@ -0,0 +1,105 @@
+        environment=data.get("environment", "production")
+    )
+
+DEPLOY_HOOKS = {
+    "capstone": "https://api.cloudflare.com/client/v4/pages/webhooks/deploy_hooks/{hook_id}",
+    "rwc": "", # TODO add/change URLS as needed
+}
+
+@router.post("/rebuild", response_model=RebuildResponse)
+async def rebuild(
+    site: str = Query(..., description="Target site to rebuild"), 
+    settings: WorkerSettings = Depends(get_settings), 
```

Ruff (0.15.11)

[warning] Unused function argument: `settings`

(ARG001)